templates/init/functions/typescript/package.lint.json

```
@@ -1,7 +1,7 @@
 {
   "name": "functions",
   "scripts": {
-    "lint": "tslint --project tsconfig.json",
+    "lint": "eslint ./src/**/*",
```

```suggestion
    "lint": "eslint src",
```

I think doing `src` is sufficient - globs are evaluated differently in various environments so it's probably safer to just say `src`